features/networking/sriov.feature

```diff
@@ -35,6 +35,8 @@ Feature: Sriov related scenarios
       | namespace   | openshift-sriov-network-operator |
       | o           | yaml                             |
     Then the step should succeed
+    And the output should contain "intel-netdevice"
+    And the output should contain "syncStatus: Succeeded"
```

Comment:
```
  And the output should contain:
    | xxx |
    | yyy |
```
which are less typing and more readable.

Reply:
updated, thanks